hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -200,7 +212,7 @@ public DBStore build() throws IOException {
       if (rocksDBOption == null) {
         rocksDBOption = getDefaultDBOptions(tableConfigs);
       }
-
+      setDBOptions(rocksDBOption);
```

Comment:
What is this line doing? Setting itself again with the same value?

Reply:
This was a mistake should have been setDbOptionsProps.